package.json

```diff
@@ -10,10 +10,12 @@
     "build": "node-gyp rebuild",
     "test": "node test/libvirt.test.js",
     "clean": "rimraf build & rimraf generated",
-    "generate": "node src/generator.js",
+    "generate": "npm run generate:all && npm run generate:format",
+    "generate:all": "node src/generator.js",
+    "generate:format": "npx clang-format -i -style=Google generated/*",
+    "check": "npm run check:libvirt_min_ver & npm run check:libvirt_api_xml",
     "check:libvirt_api_xml": "node tools/api_xml_files.js",
-    "check:libvirt_min_ver": "node tools/check_min_libvirt_ver.js",
-    "check": "npm run check:libvirt_min_ver & npm run check:libvirt_api_xml"
+    "check:libvirt_min_ver": "node tools/check_min_libvirt_ver.js"
```

Comment:
This is a small enough hunk on which it is easy to show what I meant by commit separation. One commit should be the reordering of `check` and `check:libvirt_min_ver` targets (trivial, push it right away) and yet another one should be the addition of the formatting. With bigger patches it goes very ugly very fast when you are looking at which patch broke something.

Reply:
agreed, split the patch, thanks